#PLANNING CONFIGURATIONS:
group_name: "manipulator"
max_distance: 1
n_tests: 10000

# HUMAN-AWARE COST FUNCTION CONFIGURATION
base_frame: "world"
tool_frame: "flange"
max_step_size: 0.01
object_type: "red_box"
n_object: 0
n_threads: 4
